README.md

```diff
@@ -93,10 +93,44 @@ You can also follow build statuses in https://percy.io/Homeday/homeday-blocks
 
-## Deployment
+## Contribution Guide
 
-The [CI](https://travis-ci.com/homeday-de/homeday-blocks) will deploy storybook automatically to our [showcase env](https://blocks.homeday.dev), once PR is merged to `develop` branch. 
+This project follows [forking workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow). See [project setup](#project-setup) to get started locally. That means that all code changes enter the project by PR to `develop` branch. Once you open the PR with suggested changes, the checks for `build` and `coverage` will run. If those fail, your PR needs some more work. :) 
 
+Each PR should be reviewed by at least two team members. Once reviewed and approved,it can be merged. Please follow the following convention when merging the PR: 
+
+1) Use "Squash and merge" to merge the PR. 
+2) Use a meaningful title when merging your PR. 
+3) Give title a semantic meaning through [emojis](https://gitmoji.carloscuesta.me/). Please use emoji first, then the text. 
+4) **:boom: - marks breaking changes (MAJOR version change)** 
+5) **:sparkles: - marks new feature (MINOR version change)**
+6) Everything else is consider a patch change.
+
+It is important to follow this convention, because it makes generating changelog much easier (automatic). We use [changelog github action](https://github.com/homeday-de/github-action-changelog-generator) to automatically generate the changelog. Please see the link for more details on supported conventions. 
+
+#### Examples 
+
+##### Major change
+:boom: Removed HdNotNeeded component (#32)
+
+##### Minor change
+:sparkles: Added HdAwesomeComponent (#33)
+
+##### Patch
+:wrench: Updated travis configuration and run lint on build(#34)
+
+## Version release
+
+When a new version is ready to be released, these are the steps we need to take:
+
+1) Create a release PR from `develop` to `master`
+2) Changelog will be generated during the checks. Check it to determed the release type - `MAJOR`, `MINOR` or `PATCH`.For more info, please see [contribuiton guide](#contribution-guide). If in doubt, consult with rest of the team. 
+3) Create a version bump PR to develop (use [`npm version`](https://docs.npmjs.com/cli/version.html) to make sure version is updated everywhere). This should be merged as last PR before releasing.
+4) Once version bump PR is merged to `develop` and all changes on release PR have passed, merge the release PR. 
+5) Draft a github release with corresponding version (target `master` branch), include the generated changelog, and publish it. Version naming convention is `vx.x.x`.
+6) Double check that the new version has been automatically published on [npm registry]( https://www.npmjs.com/package/homeday-blocks)
```

Comment:
I'll add a bit more explanation here about automatic push to npm. Which action does it and when exactly it's triggered. No magic. :100:  :wink: